src/iperf_api.c

```diff
@@ -2513,10 +2517,12 @@ get_results(struct iperf_test *test)
 			j_retransmits = cJSON_GetObjectItem(j_stream, "retransmits");
 			j_jitter = cJSON_GetObjectItem(j_stream, "jitter");
 			j_errors = cJSON_GetObjectItem(j_stream, "errors");
+                        j_omitted_errors = cJSON_GetObjectItem(j_stream, "omitted_errors");
 			j_packets = cJSON_GetObjectItem(j_stream, "packets");
+                        j_omitted_packets = cJSON_GetObjectItem(j_stream, "omitted_packets");
 			j_start_time = cJSON_GetObjectItem(j_stream, "start_time");
 			j_end_time = cJSON_GetObjectItem(j_stream, "end_time");
-			if (j_id == NULL || j_bytes == NULL || j_retransmits == NULL || j_jitter == NULL || j_errors == NULL || j_packets == NULL) {
+			if (j_id == NULL || j_bytes == NULL || j_retransmits == NULL || j_jitter == NULL || j_errors == NULL || j_omitted_errors == NULL || j_packets == NULL || j_omitted_packets == NULL) {
```

Comment:
There's a potential interoperability problem here...if an iperf3 with this change is doing a test against an iperf3 before this change, the older iperf3 won't have `omitted_errors` or `omitted_packets` members in the per-stream statistics structure. So this conditional will fail and cause the test to error out. I haven't figured out a good way to do backward compatibility for this yet, but I think it's somewhat important.

Reply:
Forgot about backward compatibility ...  I **re-implemented the change**, taking this into account.  When the receiver error count during the omit time is not known (because older version is used), the new `report_bw_udp_format_no_omitted_error` message format is used that shows the error count as "Unknown" (and doesn't show lost percent).

Reply:
OK great, the newest version of this PR seems to restore interoperability. Thanks!